### PR TITLE
LPS-157934

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

Fixed the error in https://github.com/liferay-core-infra/liferay-portal/pull/1071
This is for ticket https://issues.liferay.com/browse/LPS-157934, which is the preparation for ticket https://issues.liferay.com/browse/LPS-149258.
I need to use commos-io version 2.11.0 in portal-tika, and there is a limitation in using 2.8.0 in source-formatter, and that's why I will have to update this in master first.
Thanks for checking.

### CODE REVIEW

**Comment by jeffwu0724** [comment]
@catarineaguiar Can you help me look at the test result, please?

Thanks
Jeff

**Comment by catarineaguiar** [comment]
ci:test:acceptance

**Comment by catarineaguiar** [comment]
retrying because most tests were aborted

**Comment by jeffwu0724** [comment]
ci:test:acceptance

**Comment by jeffwu0724** [comment]
ci:test:acceptance

**Comment by Kvale1733** [comment]
@jeffwu0724 I found no regressions in this acceptance build:https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1906908202.

**Comment by jeffwu0724** [comment]
Thanks @Kvale1733 !   :)

**Comment by jeffwu0724** [comment]
rebased and fixed the code conflict in "lib/versions-complete.xml" by auto generate again.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

File: modules/util/portal-tools-rest-builder/bnd.bnd [hunk 1]
```diff
@@ -33,7 +33,5 @@ Main-Class: com.liferay.portal.tools.rest.builder.RESTBuilder
 	@com.liferay.portal.kernel-*.jar!/com/liferay/portal/kernel/util/*,\
 	@com.liferay.portal.tools.java.parser-*.jar!/com/liferay/portal/tools/java/parser/*.class,\
 	@com.liferay.portal.tools.java.parser-*.jar!/com/liferay/portal/tools/java/parser/util/JavaParserUtil.class,\
-	@commons-io-[0-9.]*.jar!/org/apache/commons/io/FileUtils.class,\
```
Comment (tinatian) [inline modules/util/portal-tools-rest-builder/bnd.bnd:36]: @jeffwu0724 This removal is kind of risky, as this tool is going to be used for all released versions, please pay attention to this change once this pull is merged. If it breaks anything, try bringing this back.

Tina.

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/120819

**Pull request closed without merging**